UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -172,9 +172,9 @@ public NetworkIdentity GetMasterNetID()
 	/// </summary>
 	/// <param name="cataloguePage">Index as defined via the inspector on the object.</param>
 	/// <param name="networked">Whether this change should be sent to clients, if server.</param>
-	public void ChangeSprite(int cataloguePage, bool networked = true)
+	public void ChangeSprite(int cataloguePage, bool networked = true, bool forceUpdate = false)
```

Comment:
I told you what the issue was is that, 
The state of the sprite being visible 
is a different
 state from the Sprite being set

Reply:
still here

Reply:
Nope

Reply:
![image](https://user-images.githubusercontent.com/27101670/230477102-7d2c28e5-2443-4f5b-a89d-05311918b763.png)

Reply:
![image](https://user-images.githubusercontent.com/34368774/230506114-b3057213-bb12-4a75-b8ac-a05875e48eea.png)

???????????????????????????????????????????????????????????????????????????????????